README.md

```diff
@@ -11,13 +11,29 @@ This fork is a community version of the `nupic.core` C++ repository with Python
 Our aim is to provide an actively developed successor to the nupic.core and nupic repositories by Numenta, 
 which are not actively developed anymore. 
 
+
+### Our goals
```

Comment:
I tried to make clear WHY sb. should look (use) our repo instead of the numenta/nupic.core

Please review, esp. if you agree with the goals (+-) @dkeeney @ctrl-z-9000-times